samtranslator/region_configuration.py

```diff
@@ -28,11 +28,21 @@ def is_apigw_edge_configuration_supported(cls):
     @classmethod
     def is_sar_supported(cls):
         """
-        SAR is not supported in af-south-1 at the moment.
+        SAR is not supported in some regions.
         https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/
+        https://docs.aws.amazon.com/general/latest/gr/serverlessrepo.html
 
         :return: True, if SAR is supported in current region.
         """
-        return boto3.Session().region_name not in [
-            "af-south-1",
-        ]
+
+        session = boto3.Session()
+        # get the current region
+        region = session.region_name
```

Comment:
is it possible this is `None` by any chance?

Reply:
ok, seems `region` is possible to be `None` according the code you referenced, then `region.startswith("cn")` could crash in this case.